lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Server.hs

```diff
@@ -153,7 +153,7 @@ spec = do
                 let cmd = Command
                         (commandName @t)
                         ["serve"
-                        , "--database", "/does-not-exist"
+                        , "--database", "/*"
```

Comment:
:thinking: what was the error on windows :thinking: ? Was the folder simply created ?